src/Generators/PestTestGenerator.php

```diff
@@ -376,10 +376,7 @@ protected function buildTestCases(Controller $controller): string
                 } elseif ($statement instanceof RedirectStatement) {
                     $tested_bits |= self::TESTS_REDIRECT;
 
-                    $assertion = sprintf(
-                        '$response->assertRedirect(route(\'%s\'',
-                        config('blueprint.plural_routes') ? Str::plural(Str::kebab($statement->route())) : Str::kebab($statement->route())
-                    );
+                    $assertion = sprintf('$response->assertRedirect(route(\'%s\'', $statement->route());
 
```

Comment:
Whatever route the user specifies in their _draft file_ should be used directly.